test default due fails with assertionerror

Hi,

In the last update in Archlinux (updating from 2.0.2-1 to 2.1.0-1) I got the following failure while performing the test `test_default_due2`

```
________________________________________________ test_default_due2 _________________________________________________

tmpdir = local('/tmp/pytest-of-XXXXXX/pytest-9/test_default_due20')
runner = <click.testing.CliRunner object at 0x7f9b9d3f4f60>
create = <function create.<locals>.inner at 0x7f9ba0de31e0>
default_database = <todoman.model.Database object at 0x7f9b9d3f4b00>

    @freeze_time(datetime.datetime.now())
    def test_default_due2(tmpdir, runner, create, default_database):
        cfg = tmpdir.join('config')
        cfg.write('default_due = 24\n', 'a')
    
        r = runner.invoke(cli, ['new', '-ldefault', '-dtomorrow', 'aaa'])
>       assert not r.exception
E       assert not InvalidArgsError(<class 'icu.SimpleDateFormat'>, 'format', (FakeDatetime(2003, 10, 30, 11, 45),))
E        +  where InvalidArgsError(<class 'icu.SimpleDateFormat'>, 'format', (FakeDatetime(2003, 10, 30, 11, 45),)) = <Result InvalidArgsError(<class 'icu.SimpleDateFormat'>, 'format', (FakeDatetime(2003, 10, 30, 11, 45),))>.exception

tests/test_basic.py:225: AssertionError
================================== 1 failed, 45 passed, 3 xfailed in 2.71 seconds ==================================
```

I'm not sure what's going wrong, maybe this can be helpful for you.

--- a/.travis.yml
+++ b/.travis.yml
@@ -36,6 +36,10 @@ matrix:
       python: 3.6
       after_install:
         - echo click_repl >> requirements.txt
+    - # Run tests with incompatible test dependencies:
+      python: 3.6
+      after_install:
+        - echo pyicu >> requirements.txt
 
 addons:
   apt:

--- a/.travis.yml
+++ b/.travis.yml
@@ -36,10 +36,12 @@ matrix:
       python: 3.6
       after_install:
         - echo click_repl >> requirements.txt
+      env: TOXENV=py EXTRAS=click_repl
     - # Run tests with incompatible test dependencies:
       python: 3.6
       after_install:
         - echo pyicu >> requirements.txt
+      env: TOXENV=py EXTRAS=pyicu
 
 addons:
   apt:

--- a/tests/helpers.py
+++ b/tests/helpers.py
@@ -1,0 +1,20 @@
+__all__ = [
+    'pyicu_sensitive',
+]
+
+import pytest
+
+
+def is_pyicu_installed():
+    try:
+        import icu  # noqa: F401: This is an import to tests if it's installed.
+    except ImportError:
+        return False
+    else:
+        return True
+
+
+pyicu_sensitive = pytest.mark.skipif(
+    is_pyicu_installed(),
+    reason="This test cannot be run with pyicu installed.",
+)

--- a/tests/test_cli.py
+++ b/tests/test_cli.py
@@ -11,6 +11,7 @@ from dateutil.tz import tzlocal
 from freezegun import freeze_time
 from hypothesis import given
 
+from tests.helpers import pyicu_sensitive
 from todoman.cli import cli, exceptions
 from todoman.model import Database, Todo
 
@@ -209,6 +210,7 @@ def test_move(tmpdir, runner, create):
     assert 'other_list' in result.output
 
 
+@pyicu_sensitive
 @freeze_time('2017-03-17 20:22:19')
 def test_dtstamp(tmpdir, runner, create):
     """Test that we add the DTSTAMP to new entries as per RFC5545."""
@@ -263,6 +265,7 @@ def test_default_due(
         )
 
 
+@pyicu_sensitive
 @freeze_time(datetime.datetime.now())
 def test_default_due2(tmpdir, runner, create, todos):
     cfg = tmpdir.join('config')
@@ -644,6 +647,7 @@ def test_todo_edit(runner, default_database, todo_factory):
     assert 'YARR!' in result.output
 
 
+@pyicu_sensitive
 @freeze_time('2017, 3, 20')
 def test_list_startable(tmpdir, runner, todo_factory):
     todo_factory(summary='started', start=datetime.datetime(2017, 3, 15))

--- a/tests/test_formatter.py
+++ b/tests/test_formatter.py
@@ -4,10 +4,12 @@ import pytest
 import pytz
 from freezegun import freeze_time
 
+from tests.helpers import pyicu_sensitive
 from todoman.cli import cli
 from todoman.formatters import rgb_to_ansi
 
 
+@pyicu_sensitive
 @pytest.mark.parametrize('interval', [
     (65, 'in a minute'),
     (-10800, '3 hours ago'),

